### ISSUE TITLE
No autofocus on select2 since upgrade in v5 to jquery 3.6

### ISSUE THREAD

**Issue by timbertens** [Laravel-Backpack/CRUD#4582]
# Bug report

### What I did

In v4 of backpack an older version of jQuery was used (I believe 3.4?).  In that version when using select2 fields there is an focus on the search input, so users can straight away begin to type to find the record they want

### What happened

In v5 jquery 3.6 is used and there seems to be a problem with select2 fields and focus in that version, see this PR https://github.com/select2/select2/pull/6044 that actually solves it, but does not get merged for some reason.  Maybe you guys can find some workaround in the framework that solves this issue 

When I run ```php artisan backpack:version``` the output is:

### PHP VERSION:
PHP 8.1.7 (cli) (built: Jun 25 2022 08:13:46) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.7, Copyright (c) Zend Technologies
    with Zend OPcache v8.1.7, Copyright (c), by Zend Technologies
    with Xdebug v3.1.4, Copyright (c) 2002-2022, by Derick Rethans

### LARAVEL VERSION:
v9.23.0@c4eea9060d847b5c93957b203caa8f57544a76ab

### BACKPACK VERSION:
5.3.0@6339e9c0a28ee61549fecc8cdaeb2bbddaf8b8c7

**Comment by pxpm** [comment]
Hello @timbertens 
Thanks for the report. We are aware of that issue and there's already a PR to fix it. 

I already reviewed that PR so it should be merged and tagged soon. 

I will be closing this, please re-open if needed.

Cheers

**Comment by tabacitu** [comment]
@pxpm I thought we had that fixed too, but can't find the PR. Can you? Let's prioritize merging it please.

**Comment by pxpm** [comment]
Hello @tabacitu 

This is the PR: https://github.com/Laravel-Backpack/CRUD/pull/4491

I asked @promatik for a review but later you asked @jorgetwgroup to do it, so It's just waiting one of them to give the green ligth, I've already given mine.

Cheers

**Comment by timbertens** [comment]
Thanks guys! I looked in the issues list to see if this one was already registered, but I didn't make the link with the issue I'm reporting.  You are sure we're talking about the same problem here? I see https://github.com/Laravel-Backpack/CRUD/pull/4491 is about repeatable fields, but mine is about the search box in select2 elements.

**Comment by pxpm** [comment]
Hey @timbertens 99,9% sure, or I may be missing something.
If you look at the PR code you will see that we implemented the fix for the select2 elements. 

It would be super if you can use that PR and check yourself if it fixes your issue and let us know in the PR thread if there is some use-case that we are not aware yet.

Cheers

**Comment by timbertens** [comment]
hi @pxpm should have looked at it to start with.  But you can make 100%, just tested it locally and indeed this issue is solved in PR. 

thx!

**Comment by pxpm** [comment]
Thank you @timbertens 🙏 

Very much appreciated the feedback. 

Like I said, I tested the PR too, and gived my thumbs up, so it should be "triple-checked" soon and get merged. 
We should do it carefully and surelly as it deals with very important and "globally used" crud functionality. 
Having an extra pair of eyes give us more confidence that we achieved a working solution here. 

Once again, thanks for your time here. 👍
